### PR TITLE
Added test to fix code coverage

### PR DESCRIPTION
**Pull request by KDNeufeld**
# Description

This PR includes the following proposed change(s):

The code coverage was showing missing logic from recent PRs.

image
In this case, line PaymentList line 61 was referencing a recent change - coverage needs to cover the case when keyDown is pressed, but not the Enter key (13).
image

@yousiefc, when you get a chance, can you please fix the PaymentList?  We're now getting a warning from a recent PR.
image

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

new test
yarn test

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [x] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by yousiefc** [comment]
Yes for sure! Should take a moment

**Review by yousiefc — APPROVED** [review]
sorry I didnt catch the packageId bug on my PR. Should be all good to go

**Comment by KDNeufeld** [comment]
Awesome, thanks.

File: src/frontend/efiling-frontend/src/domain/package/package-review/PaymentList.js [hunk 1]
```diff
@@ -91,18 +92,14 @@ export default function PaymentList({ payments, packageId }) {
...
                   </td>
                   <td className="text-right">
                     {payment.feeExempt
                       ? "Exempt"
                       : payment.processedAmount
-                      ? Dinero({
-                          amount: payment.processedAmount * 100,
-                        }).toFormat(format)
+                      ? formatCurrency(payment.processedAmount)
                       : "Pending"}
```
Comment (KDNeufeld) [inline src/frontend/efiling-frontend/src/domain/package/package-review/PaymentList.js:103]: Nice code-reuse!

**Pull request merged**